File: scholar-business/src/main/java/com/varun/scholar/business/data/entities1/RouteSearch.java
package com.varun.scholar.business.data.entities1;

import java.util.Date;

import javax.persistence.Entity;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class RouteSearch extends AbstractEntity {
	int routeId;
	String routeNumber;
	String vehicleNumber;
	String routeDetails;
	String driverName;
	Date lastUpdatedDate;
}

File: scholar-web/src/main/webapp/app/model/transport/route/RouteSearch.js
Ext.define('scholar.model.transport.route.RouteSearch', {
	extend: 'Ext.data.Model',
	idProperty: 'id',
	fields : [
	          {
				name : 'id',
				type : 'int'
			  },
	          {
				name : 'routeNumber',
				type : 'string'
			  },
	          {
				name : 'vehicleNumber',
				type : 'string'
	          },
	          {
				name : 'routeDetails',
				type : 'string'
	          },
	          {
				name : 'driverName',
				type : 'string'
	          },
	          {
				name : 'lastUpdatedDate',
				type : 'date',
				dateFormat : 'n/j h:ia'
	          } 
	         ]
});